### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Fernando1596** [WeaveAche/hanime-auto-coins-collector#14]
Hi guys

I never use python or java language, any idea to run this script?. any idea to make more simple just like have GUI on it?

**Comment by zip6como** [comment]
I know I'm very late.
No, as of right now, you will have to use the command prompt or terminal to use this script.
If you still need a more detailed tutorial, just reply to this comment.


Please note that I am NOT affiliated with this project in any way, I am NOT a developer of this. I am just trying to help.

**Comment by dahyunnie908** [comment]
Just the command prompt? no additional things to download? If so, please teach us, thank you!

**Comment by zip6como** [comment]
> Just the command prompt? no additional things to download?

You will need to get python and add it to path.
I have made a [tutorial](https://github.com/zip6como/hanime-auto-coins-plus/wiki/Install-Python-on-Windows) on how to do that.

Please note that I am NOT affiliated with this project in any way, I am NOT a developer of this. I am just trying to help

**Comment by dahyunnie908** [comment]
Hello! thanks for making me run the script. The only problem is running the command automatically. I can't save the task since there's no action. Again, thank you!

**Comment by dahyunnie908** [comment]
I managed to create a .bat file. It's working

@ECHO OFF
timeout 1 >nul
cd "YOUR PATH"
python AIO.py
timeout 1 >nul
exit

**Comment by dahyunnie908** [comment]
Don't know if I did it correctly since I rarely use Github but I did starred your tutorial. Again, thank you for that!

**Comment by BalajiX5** [comment]
Everything is working except the task for autorun. What should i do? Pls help if anyone is free.
Thanks for the wonderful tutorial @zip6como .

**Comment by zip6como** [comment]
> Everything is working except the task for autorun. What should i do?
I updated the autorun tutorial. You can use the new one and check if it worked.

Please note that I am NOT affiliated with this project in any way, I am NOT a developer of this. I am just trying to help

**Comment by BalajiX5** [comment]
@zip6como , I tried the updated tutorial for the autorun part. It still doesn't work. I am using win 11 64 bit.  I can collect manually, so till the last step all the points in your tutorial works. Pls Help. I would be very grateful.!!

**Comment by zip6como** [comment]
> so till the last step all the points in your tutorial works.

What do you mean by last step?

**Comment by BalajiX5** [comment]
![IMG_20230731_014842](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/7c300e84-faf3-4062-bb87-5b2d9e781c4f)

I meant the last step of your tutorial i.e is the autorun setup. 
Can you make a small video of the autorun part if possible. I don't know what I am doing wrong.

**Comment by zip6como** [comment]
I won't make a video.
Can you please tell me at wich part exaclty you are stuck?

**Comment by BalajiX5** [comment]
@zip6como 
That's the problem. I did all the steps exactly for the auto run procedure. It still doesn't run every 3 hrs. I have to do it manually by running the win Autorun.bat.  Then only it works.

**Comment by zip6como** [comment]
Can you please provide me Screenshots of the task you created in the task scheduler?

**Comment by BalajiX5** [comment]
Ok just a couple of mins.

**Comment by BalajiX5** [comment]
![1](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/9871f049-90f3-4ff1-a5d0-500a727c25aa)
![2](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/0e2dc5ba-21e6-49cc-b452-c4a944c3c78f)
![3](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/178ed858-cf4e-433e-9155-b70518b49a7a)
![4](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/62436406-3338-498a-9a83-fc5badad51cc)
![5](https://github.com/WeaveAche/hanime-auto-coins-co
![6](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/68cb3158-da0b-4d1d-8efb-b67b41d23f35)
llector/assets/125268449/f043dfc5-75fa-4575-aac8-9742be410fd6)
![7](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/37b0a16f-338a-4d07-a85d-f7a097aa1550)

**Comment by BalajiX5** [comment]
![1](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/66d2c66e-544e-4cb0-888f-d431b3edb92b)



![2](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/bb74f1ed-f349-4d07-855d-598a0a7792fb)

![3](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/1f7a2829-dede-4d02-a5fa-4d337a1be768)

![4](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/1a751faf-034f-4cab-b890-50c9d15bf2e8)

![5](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/0c5f68b8-22f0-4ad2-8ae3-6da457b4a3cb)

![6](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/efba6d3e-b855-4c54-a977-fd2cf1a38caa)

![7](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/382789d8-a383-46f8-90eb-1069e448cae3)

**Comment by BalajiX5** [comment]
@zip6como  
the above mentioned are my screenshots of task scheduler. pls help

**Comment by zip6como** [comment]
That seems about rigth.
1. Does the script work when you run the WIN-autorun.bat?
2. Are you on Desktop or Laptop?
3. Can you open the WIN-autorun.bat with a text editor and provide a screenshot?

**Comment by BalajiX5** [comment]
![Screenshot (35)](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/6b17c068-31f3-496c-b5b8-480de5c9ba29)

**Comment by zip6como** [comment]
This is really strange i have never seen a problem like this. Everything looks okay.
Are you sure you followed the last tutorial completely? are you sure it doesn't run every 3h (it should run hidden)? If yes then you can also look up on google how to run a batch script automatically, because i have ran out of ideas where the Problem could be.

**Comment by zip6como** [comment]
If even google can't assist you, you should check if the problem is caused by your system and not my script.
If nothing helps you can comment here again and i will try to further assist you.

**Comment by BalajiX5** [comment]
@zip6como 
Yes, i followed the last tutorial step by step and still didn't work every 3 hrs.I will look upon Google and try some new methods. 
Thanks for your support. 
If it still doesn't work from Google methods, I will contact you again. Thanks once again.,🙏

**Comment by zip6como** [comment]
No problem. If you still need help you should create a new issue here: https://github.com/zip6como/hanime-auto-coins-plus/issues as this is **my** repository and not this one because this one doesn't belong to me and doesn't have feautures like the autorun or the webhook.

**Comment by zip6como** [comment]
@BalajiX5
I have found the issue in my tutorial & repository
if you still can't autorun the script, check out the new tutorial.

**Comment by zip6como** [comment]
What do you mean by compile?
So you are saying that the task schedule runs once and then it stops running?

**Comment by BalajiX5** [comment]
@zip6como ,
sorry for my wrong phrasing earlier.

I manually started the WIN-autorun.bat to see if it was working. Thats why it ran once. after that it didnt even start autorun process .[ I tried putting 5 mins interval to check if it was showing in the discord webserver, still **NO**.]

I didn't try for 3 hours since it didn't start and showed no result in webhook [but shows if I do it manually when running  WIN-autorun.bat]

I downloaded the code which was recently updated and reinstalled following the whole tutorial again.

**Comment by zip6como** [comment]
Could you please manually run the task by right-clicking it and tell me if it works?

**Comment by BalajiX5** [comment]
@zip6como 
Yep, definitely works. Checked just now!!

![Screenshot (37)](https://github.com/WeaveAche/hanime-auto-coins-collector/assets/125268449/915b25c7-9699-49ab-b984-d8085c908733)

**Comment by zip6como** [comment]
So if I'm right, the script works when right-clicked on in the task scheduler, but it doesn't run automatically, correct?

**Comment by BalajiX5** [comment]
@zip6como 

Yes, what you said!

**Comment by zip6como** [comment]
I will look intro this issue in a bit, it seems weird to me, I will have to check it on my own system.

**Comment by BalajiX5** [comment]
sure!. if you are free you can also run it through anydesk on my system? if its not a problem!

**Comment by zip6como** [comment]
This is a 100% **NO** from me.
Please **DO NOT** offer random people on the internet remote access to your computer.

**Comment by BalajiX5** [comment]
Oh! i did not think too much when i mentioned it to you, sorry for that.
I just thought maybe i will change the system password after doing that!

**Comment by zip6como** [comment]
I get what you wanted to do.
But please **NEVER** offer anyone you don't know remote access.
Even if you change password.

**Comment by BalajiX5** [comment]
@zip6como 
sure man. just got a little excited i guess. Thanks for the reminder. 
Will try to be more careful from now on.

**Comment by zip6como** [comment]
Hey, don't worry, mistakes are what make us humans.
Stay safe.

**Comment by BalajiX5** [comment]
Thanks. Pls help solve this issue when you are free.!

**Comment by zip6como** [comment]
Okay, so this issue is going to require some more communication.
Could you please open a issue in **my** repository [here](https://github.com/zip6como/hanime-auto-coins-plus/issues)?

**Comment by BalajiX5** [comment]
@zip6como 
I've opened new issue.!

**Comment by zip6como** [comment]
I know I'm testing what the problem could be

**Comment by Fernando1596** [comment]
I think there nothing wrong with the code, i used it right now with self configuration for task scheduler but the timer i put 185 minutes just in case

**Comment by zip6como** [comment]
@Fernando1596 im sorry i can't understand what you are trying to Tell me

**Comment by dahyunnie908** [comment]
@zip6como bro, I was wondering why your tutorial's suddenly gone. I hope you're doing fine.

**Comment by zip6como** [comment]
> @zip6como bro, I was wondering why your tutorial's suddenly gone.

I deleted the repository
> I hope you're doing fine.

I am doing fine

@dahyunnie908

**Comment by WeaveAche** [comment]
> > @zip6como bro, I was wondering why your tutorial's suddenly gone.
> 
> I deleted the repository
> 
> > I hope you're doing fine.
> 
> I am doing fine
> 
> @dahyunnie908

Oh why delete it? It was probably helping a lot of people with little or no knowledge of python. Since I basically just wrote sort of a PoC without actually explaining how to use it properly.

**Comment by zip6como** [comment]
> Oh why delete it? It was probably helping a lot of people with little or no knowledge of python. Since I basically just wrote sort of a PoC without actually explaining how to use it properly

I made a fork of your project, but the code was really bad.
So I deleted it, but this also deleted the tutorial.

**Comment by dahyunnie908** [comment]
> Oh why delete it? It was probably helping a lot of people with little or no knowledge of python. Since I basically just wrote sort of a PoC without actually explaining how to use it properly.

Yep, my guy @zip6como really helped us back then. I was going to do this again since I got a replacement pc for the one that got broken but then the tutorial was already gone. But still, it's good that you guys are doing fine. Thought you already deactivated or something.

**Comment by zip6como** [comment]
> Yep, my guy @zip6como really helped us back then. I was going to do this again since I got a replacement pc for the one that got broken but then the tutorial was already gone. But still, it's good that you guys are doing fine. Thought you already deactivated or something.

Well since WeaveAche is back he can help, or if he gives me permission I could create a new tutorial/wiki on this repo

**Comment by WeaveAche** [comment]
> Well since WeaveAche is back he can help, or if he gives me permission I could create a new tutorial/wiki on this repo

It won't be possible for me to do it. But I've given public write access to the wiki of this repository. You can add the tutorial there if you want, then we can link it from the readme.

**Comment by zip6como** [comment]
> It won't be possible for me to do it. But I've given public write access to the wiki of this repository. You can add the tutorial there if you want, then we can link it from the readme.

I might do this in the future. However, I don't think you should keep write permissions public, since anyone can just write anything they want in it.

**Comment by XKIRMADAX** [comment]
can someone just run this file in their computer idont know python and script doesnt run at all  my mail is 'qegafywi@citmo.net' and pass is '0p0p0p0p' this isnt a burner mail so no worry

**Comment by MilesUpshur420** [comment]
For anyone still searching, I found a solution, although I am not sure if it would work for the rest of you.

The solution is to add the location of the project, i.e., "C:\Users\User\VisualStudioCode\hanime_bot\hanime-auto-coins-collector" in the **Start in (optional):** part of the **Edit Action** in **Properties -> Actions** of the scheduled task.

Also, here is the code of the bat file that I am using:

_WIN-autorun.bat_
```
echo %date% %time% >> C:\Users\User\Documents\output.txt
python.exe getcoins.py >> C:\Users\User\Documents\output.txt
```

Please make sure that you are changing the directories mentioned here as per your needs.
